code/modules/mob/living/simple_animal/abnormality/waw/snow_whites_apple.dm

```diff
@@ -0,0 +1,264 @@
+/mob/living/simple_animal/hostile/abnormality/F0442
+	name = "Snow Whites Apple"
+	desc = "An abnormality taking form of a tall humanoid with an apple for a head."
+	icon = 'ModularTegustation/Teguicons/32x64.dmi'
+	icon_state = "snowwhitesapple_inert"
+	icon_living = "snowwhitesapple_inert"
+	icon_dead = "snowwhitesapple_dead"
+	var/icon_aggro = "snowwhitesapple_active"
+	maxHealth = 1600
+	health = 1600
+	damage_coeff = list(BRUTE = 0.5, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1.0, BLACK_DAMAGE = 0, PALE_DAMAGE = 1.5)
+	ranged = TRUE
+	a_intent = "help"
+	melee_damage_lower = 40
+	melee_damage_upper = 52
+	rapid_melee = 0.5
```

Comment:
Not 100% sure if this works, just a heads up.

Reply:
I problobly should remove the melee damage thing since she attacks solely through a proc on her roots.